samples/deployment/README.md

```diff
@@ -0,0 +1,35 @@
+# Forgeops Deployment
+
+### Prerequisites
+This guide assumes you have access to Kubernetes cluster and you are able to
+do deployments. Other requirements are:
+ - You have a namespace
+ - Helm is installed (Kubernetes package manager)
+ - Ingress is deployed and you can access cluster endpoint
+
+> You can achive all of the above by running the forgeops/bin/gke-up.sh scipt.
+
+#### Deployment Sizing
+
+In `size[cluster-size]/amster.yaml`, you need to modify sedFilter
+to match your enviroment. Example follows.
+
+```
+global:
+  git:
+    sedFilter: "-e s/benchmark.example.com/[NAMESPACE].[DOMAIN]/"
+```
+
+You can also modify resource related properties.  Keeping the default is
+recommended otherwise they will change outcome of benchmark significantly.
```

Comment:
sugg: ...is recommended so that you do not change the outcome of the benchmark...